main/templates/main/list.html

```diff
@@ -28,12 +27,20 @@ <h1>File List</h1>
                 <a href="{{ file.download_url }}" class="btn btn-secondary btn-sm text-success" target="_blank">Download</a>
             </td>
             <td>
-                <a href="{% url 'delete' file_id=file.id %}" class="btn btn-secondary btn-sm text-danger">Delete</a>
+              <form action="{% url 'openhumans:delete_file' %}" method="post">{% csrf_token %}
```

Comment:
Tip: in Firefox and Chrome there's page inspectors you can use to see the final HTML that is being produced -- you should use these! :smile:

One thing you'll see is that the CSRF token is represented as something like this...
`<input name="csrfmiddlewaretoken" value="SecretTokenValueHere" type="hidden">`